flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/FlinkUtils.java

```diff
@@ -172,7 +70,7 @@ public static Configuration loadConfiguration(String confDir) {
         return configuration;
     }
 
-    private static String createTempFile(Pod podTemplate) throws IOException {
+    public static String createTempFile(Pod podTemplate) throws IOException {
```

Comment:
We could move this to the FlinkConfigbuilder and keep as private

Reply:
Moved in a02f7185edfdac9e71e470f46b69d5f89c4dacab